libraries/DataFlash/LogStructure.h

```diff
@@ -714,7 +716,7 @@ Format characters in the format string for binary log messages
     { LOG_RSSI_MSG, sizeof(log_RSSI), \
       "RSSI",  "Qf",     "TimeUS,RXRSSI" }, \
     { LOG_BARO_MSG, sizeof(log_BARO), \
-      "BARO",  "Qffcf", "TimeUS,Alt,Press,Temp,CRt" }, \
+      "BARO",  "Qffcfff", "TimeMS,Alt,Press,Temp,CRt,Corr,GpsOH" }, \
```

Comment:
you've relabeled TimeUS to TimeMS. mistake? Please remove the two instances of that.